src/TinkerServiceProvider.php

```diff
@@ -27,9 +27,11 @@ public function register()
 
         $this->commands(['command.tinker']);
 
-        $this->publishes([
-            __DIR__.'/../config/tinker.php' => $this->app['path.config'].DIRECTORY_SEPARATOR.'tinker.php',
-        ]);
+        if ($this->app->runningInConsole()) {
```

Comment:
I actually don't think config is even needed here. I'd say just force the first setting on, and the second setting off.